server.go

```diff
@@ -743,26 +755,42 @@ func (sp *serverPeer) OnGetHeaders(_ *peer.Peer, msg *wire.MsgGetHeaders) {
 	sp.QueueMessage(&wire.MsgHeaders{Headers: blockHeaders}, nil)
 }
 
-// OnGetCFilter is invoked when a peer receives a getcfilter bitcoin message.
-func (sp *serverPeer) OnGetCFilter(_ *peer.Peer, msg *wire.MsgGetCFilter) {
-	// Ignore getcfilter requests if not in sync.
+// OnGetCFilters is invoked when a peer receives a getcfilters bitcoin message.
+func (sp *serverPeer) OnGetCFilters(_ *peer.Peer, msg *wire.MsgGetCFilters) {
+	// Ignore getcfilters requests if not in sync.
 	if !sp.server.syncManager.IsCurrent() {
 		return
 	}
 
-	filterBytes, err := sp.server.cfIndex.FilterByBlockHash(&msg.BlockHash,
-		msg.FilterType)
+	hashes, err := sp.server.chain.HeightToHashRange(int32(msg.StartHeight),
+		&msg.StopHash, wire.MaxGetCFiltersReqRange)
```

Comment:
See comment below in the `wire` package about the value used here as the last arg.